includes/general.php

```diff
@@ -2402,6 +2404,7 @@ function pods_no_conflict_on( $object_type = 'post', $object = null ) {
 				$no_conflict['filter'] = array_merge( $no_conflict['filter'], array(
 					array( 'add_post_metadata', array( PodsInit::$meta, 'add_post_meta' ), 10, 5 ),
 					array( 'update_post_metadata', array( PodsInit::$meta, 'update_post_meta' ), 10, 5 ),
+					array( 'update_post_metadata_by_id', array( PodsInit::$meta, 'update_post_meta_by_id' ), 10, 4 ),
```

Comment:
* :no_entry_sign: Short array syntax must be used to define arrays
* :no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 4952f4347b446f7b170387dbb474210e</sub>